xdotool is under user/ but even with it moved to main/ it fails due to timeout reaching kded6 over D-Bus

--- a/libkmpris/autotests/CMakeLists.txt
+++ b/libkmpris/autotests/CMakeLists.txt
@@ -26,10 +26,3 @@ set_tests_properties(
         ENVIRONMENT "LC_ALL=en_US.UTF-8;LD_PRELOAD=${LIBASAN_PATH};CMAKE_BINARY_DIR=${CMAKE_BINARY_DIR}/bin;QMLTEST_PATH=$<TARGET_FILE:${QT_CMAKE_EXPORT_NAMESPACE}::qmltestrunner>"
 )
 
-find_program(xdotool_PATH xdotool REQUIRED)
-find_program(kglobalacceld_PATH kglobalacceld PATHS ${KDE_INSTALL_FULL_LIBEXECDIR} REQUIRED)
-add_test(NAME mediakeystest
-    COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/mediakeystest.py ${KDE_INSTALL_FULL_LIBEXECDIR}/kglobalacceld
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../../appiumtests
-)
-set_tests_properties(mediakeystest PROPERTIES RUN_SERIAL ON)
